migration.md

```diff
@@ -433,3 +433,4 @@ SFML 3 includes various smaller changes that ought to be mentioned.
 * Added new `sf::CoordinateType` enumeration to `sf::RenderStates::RenderStates`
 * Removed `sf::Vertex` constructors in favor of aggregate initialization
 * Renamed `sf::Mouse::Button::XButton1` and `sf::Mouse::Button::XButton2` enumerators to `sf::Mouse::Button::Extra1` and `sf::Mouse::Button::Extra2`
+* Removed `sf::Vector2<T>::UnitX` and `sf::Vector2<T>::UnitY`, use `sf::Vector2<T>{1, 0}` and `sf::Vector2<T>{0, 1}` instead
```

Comment:
These constants never existed in SFML 2

Reply:
Should have checked instead of assuming. Will fix!

BTW, they were added here:
- https://github.com/SFML/SFML/commit/70eeba5067d02b8e617989c90311a866a7bacd9f